lib/validate.js

```diff
@@ -93,6 +94,8 @@ module.exports = function makeValidatorMiddleware (middleware, schema, opts) {
         strict: opts.strict === true ? opts.strict : false
       })
       addFormats(ajv)
+
+      if (opts.keywords) { addKeywords(ajv, opts.keywords) }
```

Comment:
Is there a reason not to always load this? Is it a perf hit?

Reply:
Needed only when custom keywords are used in api definition to validate request.

Its not a perf hit.